src/google/protobuf/util/internal/utility.cc

```diff
@@ -353,21 +353,15 @@ bool IsMap(const google::protobuf::Field& field,
              google::protobuf::Field_Cardinality_CARDINALITY_REPEATED &&
          (GetBoolOptionOrDefault(type.options(), "map_entry", false) ||
           GetBoolOptionOrDefault(type.options(),
-                                 "google.protobuf.MessageOptions.map_entry", false) ||
-          GetBoolOptionOrDefault(type.options(),
```

Comment:
this was due to the different package name we use in google. One of the symbol is actually the internal code names in google, and we converted to google.protobuf duration opensource release. Please revert this.